sdk/storage/azure-storage-queue/azure/storage/queue/_queue_utils.py

```diff
@@ -0,0 +1,170 @@
+# -------------------------------------------------------------------------
```

Comment:
consider refactoring this file into logically separated entities.

Reply:
Agreed - We have an issue open for this: https://github.com/Azure/azure-sdk-for-python/issues/6030